src/modinfo.c

```diff
@@ -1520,6 +1553,11 @@ sr_module_oper_data_update(struct sr_mod_info_mod_s *mod, const char *orig_name,
             return err_info;
         }
 
+        /* check whether we need any implicit data, or it can be skippped */
+        if ((err_info = sr_module_oper_data_implicit_skip(mod, conn, get_oper_opts, &skip_implicit_data)) || skip_implicit_data) {
+            return err_info;
+        }
+
         /* add any missing NP containers in the data */
         if ((err_info = sr_lyd_new_implicit_module(data, mod->ly_mod, LYD_IMPLICIT_NO_DEFAULTS, NULL))) {
```

Comment:
switching this to `sr_lyd_new_implicit_module()` caused `test_oper_pull` to fail, and I couldn't make sense of why.